SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -46,6 +47,19 @@ UWorld* GetAnyGameWorld()
 
 } // anonymous namespace
 
+// Disable local deployments from starting and open map
+bool SetUp()
+{
+	const auto Settings = GetMutableDefault<USpatialGDKEditorSettings>();
+	const auto OldBAutoStartLocalDeployment = Settings->bAutoStartLocalDeployment;
+	Settings->bAutoStartLocalDeployment = false;
+	UE_LOG(LogTemp, Warning, TEXT("Settings->bAutoStartLocalDeployment set to %d"), Settings->bAutoStartLocalDeployment)
+
+	AutomationOpenMap(SpatialConstants::EMPTY_TEST_MAP_PATH);
+
+	return OldBAutoStartLocalDeployment;
```

Comment:
If this only applies to AutomationOpenMap it could be set back to the previous value here?